smtp.go

```diff
@@ -144,14 +144,12 @@ type smtpSender struct {
 
 func (c *smtpSender) Send(from string, to []string, msg io.WriterTo) error {
 	if err := c.Mail(from); err != nil {
-		if err == io.EOF {
```

Comment:
There's an important distinction in error handling here: Gomail shouldn't make any attempts to redial unless it was _explicitly_ an EOF. Any failure should instead be handled by the caller.

@arundudeemmt You mentioned in #104 that [QOR](https://github.com/qor/qor) relies on Gomail. Could you link me to where in the codebase it's used? I'd recommend extending _that_ project to account for errors as it isn't always the best choice to simply redial without any additional work.

Reply:
some time there are error which are not EOF too and requires a redial like write: broken pipe .For broken pipe i was not able to find any standard error codes from smtp .Please suggest a way to fix the same .